kernel/primred.ml

```diff
@@ -322,6 +322,9 @@ struct
       | Float64.FLt -> E.mkFLt env
       | Float64.FGt -> E.mkFGt env
       | Float64.FNotComparable -> E.mkFNotComparable env)
+    | Float64equal ->
+      let f1, f2 = get_float2 evd args in
+      E.mkBool env (Float64.equal f1 f2)
```

Comment:
I can't find this function. Did you forget to commit a file or am I just going blind?

Reply:
Oops, no, sorry. It is in `float64_common.ml`.

Reply:
It's in `kernel/float64_common.ml` (used by the kernel from the beginning of primitive fp, nothing new)